### DISCUSSION TITLE
On macOS (arm64) Tectonic 0.15.0 insists on writing to $HOME/Library/Caches/Tectonic after setting $TECTONIC_CACHE_DIR

### DISCUSSION THREAD

**Discussion by anjos** [tectonic-typesetting/tectonic#1242] [General]
The title says it all.  Is this to be expected or is it an issue?  For info, I'm using the build from conda-forge.

Reproducing:

1. Move or remove `$HOME/Library/Caches/Tectonic`
2. Re-create that directory and remove permissions `mkdir $HOME/Library/Caches/Tectonic && chmod u-w $HOME/Library/Caches/Tectonic`
3. `export TECTONIC_CACHE_DIR=.cache-tectonic`
4. Compile a document in the current directory

Output is:

```
export TECTONIC_CACHE_DIR=.cache-tectonic && tectonic -X compile test.tex
note: "version 2" Tectonic command-line interface activated
error: Permission denied (os error 13)
caused by: Permission denied (os error 13)
``` 

If I enable writing on `$HOME/Library/Caches/Tectonic`, the problem goes away, however Tectonic still writes a file inside that location:

```sh
tree ~/Library/Caches/Tectonic
/Users/andre/Library/Caches/Tectonic
└── formats
    └── 6ffe055852f8faf66c0acbe1a7fb27f87b869a90bad1204f3bf4d9683f597c7c-latex-33.fmt

2 directories, 1 file
```

**Comment by bennof** [comment]
Hi, 
Using a relative path like `export TECTONIC_CACHE_DIR=.cache-tectonic` is likely causing the issue. If not, its an os artifact.

**Comment by anjos** [comment]
I confirm this is gone on 0.16.x and now the caching works as it is supposed to be.
